doc/api/tls.md

```diff
@@ -876,6 +876,30 @@ socket.on('end', () => {
 });
 ```
 
+Passing in `net.Socket` will start a new instance.
+To upgrade an existing instance of `net.Socket` to a 
```

Comment:
Ah sorry, and thank you for pointing it out! :smile: Thanks so much for all of your help, this was my first open source PR and you were wonderful to work with.